### DISCUSSION TITLE
avalonia怎么实现DataTrigger

### DISCUSSION THREAD

**Discussion by Ha-rlan** [BYJRK/DotNet-Discussions#384] [其他框架]
如题，现在写log ui，想要根据不同的消息类型显示对应的消息图标，但是直接使用Converter转好像只会直接显示对应的unicode，所以想要尝试一下使用DataTrigger来实现一下看看可不可行。但是查了一下发现貌似没有Trigger,不知道该怎么实现呢？
```xaml
<TextBlock Text="{Binding Direction, Converter={converter:DirectionToFontConverter}}" FontFamily="{StaticResource IconFont}"/>
```

**Comment by Ha-rlan** [comment]
或者有没有其他的实现思路啊，目前把TextBlock的Text直接写死是有图标出来的

**Reply by BYJRK** [reply]
搞两个Class，比如ClassA和B，分别为TextBlock指定两种Text。然后Classes.ClassA去绑定Data
